stage/configuration/test_directory_origin.py

```diff
@@ -570,9 +571,40 @@ def test_directory_origin_configuration_max_line_length(sdc_builder, sdc_executo
 
 
 @pytest.mark.parametrize('data_format', ['JSON'])
-@pytest.mark.skip('Not yet implemented')
-def test_directory_origin_configuration_max_object_length_in_chars(sdc_builder, sdc_executor, data_format):
-    pass
+def test_directory_origin_configuration_max_object_length_in_chars(sdc_builder, sdc_executor, data_format,
+                                                                   shell_executor, file_writer):
+    """Check if direcotory origin honours `maximum object length in character` configuration.
+        """
+    files_directory = os.path.join('/tmp', get_random_string())
+    file_name = 'max_char_len.json'
+    json_data = get_json_data()
+    file_content = "\n".join([json.dumps(d) for d in json_data])
```

Comment:
```suggestion
    file_content = '\n'.join([json.dumps(d) for d in json_data])
```

Also, replace `d` with something more descriptive of what each `d` actually is.

Reply:
Done